cdap-watchdog/src/main/java/co/cask/cdap/logging/plugins/RollingLocationLogAppender.java

```diff
@@ -273,4 +283,12 @@ public String getDirPermissions() {
   public void setDirPermissions(String dirPermissions) {
     this.dirPermissions = dirPermissions;
   }
+
+  public void setFileCloseInterval(long fileCloseInterval) {
```

Comment:
I have added it in sample xml and not here because this is logback config and we have not added javadocs for it.